src/ui/components/App.css

```diff
@@ -79,6 +79,9 @@ h2 {
 #command-button-pick.active::before {
   background-color: var(--primary-accent);
 }
+#command-button-pick.errored::before {
+  background-color: var(--theme-icon-error-color);
```

Comment:
This color looks pink when it should be red. I don't like that, but I think this CSS var is _probably_ the one we should be using.

I think "errors" should always be red and the pinkish red color should be used for _current time_

cc @jonbell-lot23